packages/mgt/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -647,6 +718,22 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     }
   }
 
+  /**
+   * Send a chat message to the user from the quick message input.
+   *
+   * @protected
+   * @returns {void}
+   * @memberof MgtPersonCard
+   */
+  protected sendQuickMessage(): void {
+    const message = this._chatInput.trim();
+    if (!message || !message.length) {
+      return;
+    }
+
+    this.chatUser(message);
```

Comment:
This will open teams and put the message in a chat instead of actually sending them... I wonder if we should make this a real call to send a message

Reply:
We can't actually send a message, only an authorized bot can do that. This is the best we can get :/